extensions/dbt-documentation/package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
nitpicking here, but the command together with the extension name look a bit verbose. I would suggest maybe call the extension title "dbt" and then the command title "Search Documentation" with the subtitle "dbt".
<img width="1000" alt="Raycast 2022-07-15 at 12 40 41" src="https://user-images.githubusercontent.com/12091319/179207884-dbff52b5-0973-4a5d-8ce4-e0c287922b4c.png">